### ISSUE TITLE
Discretizing Continuous Variables into the Codebook

### ISSUE THREAD

**Issue by malpus** [accord-net/framework#270]
One of the main improvements of C4.5 over ID3 is it's ability to handle continuous variables. In this package, you need to create a codebook which is then made into a decision variable which is then made into a decision tree. How do you use non-categorical parameters into the codebook.

**Comment by cesarsouza** [comment]
Hi there,

Actually, it is not necessary to create a codebook in order to use the C4.5 learning algorithm in the framework.  I will add an example to the documentation on how this can be accomplished. In short, it suffices to specify the array of DecisionVariable's manually, specifying the name of each variable and whether they are continuous or discrete. For continuous variables, it is not necessary to specify their range.

For example, in order to create a decision tree to handle the continuous features of Fisher's Iris data set, one could write:

```
            DecisionVariable[] features =
            {
                new DecisionVariable("sepal length", DecisionVariableKind.Continuous), 
                new DecisionVariable("sepal width", DecisionVariableKind.Continuous), 
                new DecisionVariable("petal length", DecisionVariableKind.Continuous), 
                new DecisionVariable("petal width", DecisionVariableKind.Continuous), 
            };

            // Now, we can finally create our tree for the 3 classes:
            var tree = new DecisionTree(inputs: features, classes: 3);
```

Regards,
Cesar

**Comment by cesarsouza** [comment]
Fixed on release 3.3.0.
